apps/accounts/views.py

```diff
@@ -149,9 +154,13 @@ def post(self, request, format=None):
             payload = GenerateToken.verify_and_get_payload(dummy_token)
             # print(payload)
         except InvalidToken as e:
-            return Response({"errors": {"token":str(e)}}, status=status.HTTP_401_UNAUTHORIZED)
+            return Response(
+                {"errors": {"token": str(e)}}, status=status.HTTP_401_UNAUTHORIZED
```

Comment:
create common response.py 
where you can return errors and proper response by passing msg variable.